awx/ui_next/src/screens/Template/shared/PlaybookSelect.jsx

```diff
@@ -52,23 +53,43 @@ function PlaybookSelect({ projectId, isValid, field, onBlur, onError, i18n }) {
     }
   }, [error, onError]);
 
-  const isDisabledData = [
-    {
-      value: field.value || '',
-      label: field.value || '',
-      key: 1,
-      isDisabled: true,
-    },
-  ];
+  const renderOptions = opts => {
+    return opts.map(option => (
+      <SelectOption
+        key={option.key}
+        aria-label={option.label}
+        value={option.value}
+      >
+        {option.label}
+      </SelectOption>
+    ));
+  };
+
   return (
-    <AnsibleSelect
+    <Select
+      variant={SelectVariant.typeahead}
+      onToggle={() => {
+        helpers.setTouched();
+        setIsExpanded(!isExpanded);
+      }}
+      onSelect={(event, selection) => {
+        setIsExpanded(false);
+        onChange(selection);
+      }}
+      placeholderText={i18n._(t`Choose a playbook`)}
+      onClear={() => {
+        helpers.setTouched();
+        onChange('');
+      }}
+      selections={field.value}
+      createText={i18n._(t`Create`)}
```

Comment:
We will need to include the prop below. Otherwise this string will not be translated. I will take a look in the other places that we use the Select component also.

```jsx
      noResultsFoundText={i18n._(t`No results found`)}
```

![image](https://user-images.githubusercontent.com/9053044/100929701-0d7e3800-34b6-11eb-81c4-15d298afeecd.png)